charts/hatchet-ha/values.yaml

```diff
@@ -204,9 +212,22 @@ postgres:
     service:
       ports:
         postgresql: 5432
+    resources:
+      limits:
+        cpu: 2000m
+        memory: 4Gi
+      requests:
+        cpu: 1000m
+        memory: 2Gi
```

Comment:
Why are we setting this? It's not clear these limits/requests will fit on a cluster, it's highly dependent on node types. If someone is quickly testing the chart locally this might never provision